sdk/typescript/src/cryptography/publickey.ts

```diff
@@ -1,136 +1,82 @@
 // Copyright (c) 2022, Mysten Labs, Inc.
 // SPDX-License-Identifier: Apache-2.0
```

Comment:
this file is repurposed to an interface `PublicKey` implemented by `Ed25519PublicKey` and `Secp256k1PublicKey`